modules/account-lib/test/unit/coin/dot/transactionBuilder/transferBuilder.ts

```diff
@@ -114,7 +114,7 @@ describe('Dot Transfer Builder', () => {
       should.deepEqual(txJson.tip, 0);
       should.deepEqual(txJson.transactionVersion, txVersion);
       should.deepEqual(txJson.chainName, chainName);
-      should.deepEqual(txJson.eraPeriod, 64);
+      should.deepEqual(txJson.eraPeriod, 4);
```

Comment:
polkadot library had a bug where imortal transactions were defaulting to 64 when it shouldn't have. The latest polkaodt lib update fixes that so fixing this test so it passes with the correct eraPeriod.